packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -116,14 +124,16 @@ export default util.createRule<Options, MessageIds>({
           nodeType = `${methodDefinition.kind} property accessor`;
           break;
       }
-      if (check === 'off') {
-        return;
-      }
 
       const methodName = util.getNameFromClassMember(
         methodDefinition,
         sourceCode,
       );
+
+      if (check === 'off' || exceptMethods.includes(methodName)) {
```

Comment:
`Set` uses `has` instead of `includes`
```suggestion
      if (check === 'off' || exceptMethods.has(methodName)) {
```

Reply:
Done!